cache/run_redis.sh

```diff
@@ -5,6 +5,8 @@ set -x
 
 if [ -f  ../../valkey/src/valkey-server ]; then
     ../../valkey/src/valkey-server ./cache.conf
-else
+elif [ -f ../../redis/src/redis-server ]; then
     ../../redis/src/redis-server ./cache.conf
+else
+    /usr/bin/redis-server ./cache.conf
```

Comment:
I'm not totally against that, but I always use the latest version of redis (or valkey now), and that includes the latest commands. Using the version from the packages often gives you a very old version that may or may not work.

Can you print a message saying that it is unsupported and may cause runtime issues?